makefiles/tools/jlink.inc.mk

```diff
@@ -9,3 +9,5 @@ FFLAGS ?= flash $(FLASHFILE)
 DEBUGGER_FLAGS ?= debug $(ELFFILE)
 DEBUGSERVER_FLAGS ?= debug-server
 RESET_FLAGS ?= reset
+
+JLINK_SERIAL ?= $(DEBUG_ADAPTER_ID)
```

Comment:
This should be exported if it's used in `jlink.sh`

Reply:
Thanks for spotting this @aabadie, you are right. I'm thinkking of the best place to export. Will adddress and fix the test procedure.